thirdeye/thirdeye-frontend/app/pods/components/entity-filter/component.js

```diff
@@ -16,8 +16,8 @@
  * @exports entity-filter
  */
 
-/* global moment */
 import Ember from 'ember';
+import _ from 'lodash';
```

Comment:
👍

Reply:
hmm. why is lodash already accessible without explicitly importing it?

Reply:
I thought so too. I think it might be globally accessible, but eslint is complaining about it.